content/en/docs/hertz/tutorials/toolkit/toolkit.md

```diff
@@ -0,0 +1,1019 @@
+---
+title: 'hz command line toolkit usage'
+date: 2022-07-15
+weight: 1
+description: >
+---
+
+hz is a command line tool provided by the Hertz framework for generating code. Currently, hz can generate scaffolding for Hertz projects based on thrift and protobuf's IDL.
+
+## Install
+
+1. Make sure the `GOPATH` environment variable has been defined correctly (eg `export GOPATH=~/go`) and add `$GOPATH/bin` to the PATH environment (eg `export PATH=$GOPATH/bin:$PATH`); do not set a directory in which you have no read and write permission as `GOPATH`.
+2. Install hz:
+
+```bash
+go install github.com/cloudwego/hertz/cmd/hz@latest
+```
+
+3. Verify that the installation was successful `hz -v`, if the following version message is displayed, the installation was successful
+
+```console
+hz version v0.1.0
+```
+
+**Note**，Since hz creates soft links to its own binaries, make sure that the installation path of hz has writable permissions.
+
+## Operating mode
+
+To generate code using thrift or protobuf IDL, The corresponding compiler needs to be installed: [thriftgo](https://github.com/cloudwego/thriftgo) or [protoc](https://github.com/protocolbuffers/protobuf/releases).
+
+The code generated by hz, part of it is generated by the underlying compiler (usually about the structure defined in IDL), and the other part is the user-defined routing, method and other information in IDL. The user can run the code directly.
+
+In terms of execution flow, when hz uses thrift IDL to generate code, hz calls thriftgo to generate the go structure code and executes itself as a plug-in to thriftgo (named thrift-gen-hertz) to generate the rest of the code. This is also true when used with the protobuf IDL.
```

Comment:
"hz calls thriftgo to generate the go structure code and executes itself as a plug-in to " -> "hz calls thriftgo to generate the go struct code and executes itself as a plugin to "

Reply:
fixed.